monkey/monkey_island/cc/resources/root.py

```diff
@@ -19,8 +20,8 @@ def get(self, action=None):
             action = request.args.get("action")
 
         if not action:
-            return make_response(204)
+            return {}, HTTPStatus.NO_CONTENT
         elif action == "is-up":
             return {"is-up": True}
         else:
-            return make_response(400, {"error": "unknown action"})
+            return {"error": "unknown action"}, HTTPStatus.BAD_REQUEST
```

Comment:
Can you provide an explanation of the root cause and how this resolves it? I just don't understand it.

Reply:
@shreyamalviya this is the root cause. I suppose that when we make a response with an integer it throws `TypeError`

![image](https://user-images.githubusercontent.com/15820737/196387916-19cebc05-326a-43c3-93de-00d8bd0b800e.png)

Reply:
How does that result in an empty log file?

Reply:
This PR is just for fixing the runs of the blackbox tests. I am still working on the empty log files. Sorry for the wrong branch name.